homeassistant/components/uptimerobot/entity.py

```diff
@@ -75,4 +75,6 @@ def monitor_available(self) -> bool:
     @property
     def available(self) -> bool:
         """Returtn if entity is available."""
+        if not self.coordinator.last_update_success:
```

Comment:
Call super instead.
```py
return super().available and self.monitor is not None
```

https://github.com/home-assistant/core/blob/7d29eb282bf1670692d6bc4d38d76dec22029279/homeassistant/helpers/update_coordinator.py#L310-L313

Reply:
With the management of the devices that were added, self.monitor will never be None, so this property should be removed instead.

Reply:
https://github.com/home-assistant/core/pull/54259